cmd/install.go

```diff
@@ -220,19 +220,20 @@ func (h *hammer) writeInstallerConfig(machine *models.V1MachineResponse, rootUUi
 	}
 
 	y := &api.InstallerConfig{
-		Hostname:      *alloc.Hostname,
-		SSHPublicKey:  sshPubkeys,
-		Networks:      alloc.Networks,
-		MachineUUID:   h.spec.MachineUUID,
-		Password:      h.spec.ConsolePassword,
-		Console:       console,
-		Timestamp:     time.Now().Format(time.RFC3339),
-		Nics:          h.onlyNicsWithNeighbors(machine.Hardware.Nics),
-		VPN:           alloc.Vpn,
-		Role:          *alloc.Role,
-		RaidEnabled:   raidEnabled,
-		RootUUID:      rootUUiD,
-		FirewallRules: alloc.FirewallRules,
+		Hostname:         *alloc.Hostname,
+		SSHPublicKey:     sshPubkeys,
+		Networks:         alloc.Networks,
+		MachineUUID:      h.spec.MachineUUID,
+		Password:         h.spec.ConsolePassword,
+		Console:          console,
+		Timestamp:        time.Now().Format(time.RFC3339),
+		Nics:             h.onlyNicsWithNeighbors(machine.Hardware.Nics),
+		VPN:              alloc.Vpn,
+		Role:             *alloc.Role,
+		RaidEnabled:      raidEnabled,
+		RootUUID:         rootUUiD,
+		FirewallRules:    alloc.FirewallRules,
+		FirewallDistance: 0,
```

Comment:
This was the idea behind it and it works pretty good in the code.
https://github.com/metal-stack/metal-networker/pull/98#issuecomment-2459832408

Reply:
https://github.com/metal-stack/firewall-controller/blob/d23ded771bcf0e8a96d2699e571ec8cf66c05d76/pkg/network/network.go#L73

Reply:
A High distance like 10 should be enough?

Reply:
Highest value used by the FCM is 8: https://github.com/metal-stack/firewall-controller-manager/blob/91feb1f750de70a2008b81f3bf0dda41aa4be13f/api/v2/types_firewallset.go#L15

Reply:
> In general I am not sure if this is required in the metal-hammer at all, but if we want to do it here, too, then the distance should be high because only then newly created firewalls do not attract traffic immediately.

It's good to have that already in the metal-hammer. Otherwise if we roll firewalls and the new one starts up it has the smallest distance until the firewall-controller starts. This might disturb outgoing traffic flows.

If firewalls start with a high distance they attract no traffic and the firewall-controller-manager decides to configure a lower distance once the new one is configured and the old firewall is destroyed.